packages/fx-core/src/plugins/solution/fx-solution/arm.ts

```diff
@@ -603,9 +604,21 @@ export async function deployArmTemplates(ctx: SolutionContext): Promise<Result<v
       );
     }
   } catch (error) {
-    result = err(
-      returnUserError(error, SolutionSource, SolutionError.FailedToDeployArmTemplatesToAzure)
```

Comment:
@dilin-MS can you help recall why we set the error to user error here? If no special reason I will update the code to set it as system error

Reply:
discussed offline, will change the error type to system error